app/helpers/admin_access_helper.rb

```diff
@@ -3,26 +3,33 @@ def access_facility_count(available)
     "#{available} #{"facility".pluralize(available)}"
   end
 
-  def access_checkbox(form, name, resource, page: :new, checked_fn: -> { false })
-    return access_resource_label(resource) if page.eql?(:show)
+  def access_checkbox(name, resource, page: :new, checked_fn: -> { false })
+    case page
+      when :show
+        content_tag(:div, class: "form-check__show") { access_resource_label(resource) }
```

Comment:
It took me a while to grok what's going on in here. Why overload this method if we're not returning a checkbox in this case? Should we use two different methods?

Reply:
We can do that, I stuck it in here to keep the views simple, if I do separate methods, the check will go in the view, since the partials are shared.

Reply:
I have an idea to make this clearer, let me give it a shot.